benchto-it/src/test/java/io/trino/benchto/integrationtest/BenchtoTrinoIntegrationTest.java

```diff
@@ -0,0 +1,161 @@
+ */
+package io.trino.benchto.integrationtest;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import io.trino.benchto.driver.BenchmarkProperties;
+import io.trino.benchto.driver.DriverApp;
+import io.trino.benchto.driver.execution.ExecutionDriver;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.util.concurrent.Uninterruptibles.sleepUninterruptibly;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = DriverApp.class)
+@org.springframework.boot.test.IntegrationTest("executionSequenceId=BEN_SEQ_ID")
+public class BenchtoTrinoIntegrationTest
+{
+    static PostgreSQLContainer<?> postgres;
+    static GenericContainer<?> service;
+    static GenericContainer<?> trino;
+
+    @Autowired
+    private ExecutionDriver executionDriver;
+
+    @Autowired
+    private BenchmarkProperties benchmarkProperties;
+
+    @Autowired
+    protected ApplicationContext context;
+
+    @BeforeClass
+    public static void setup()
+    {
+        Network network = Network.newNetwork();
+        startBenchtoService(network);
+        startTrino(network);
+    }
+
+    @Test
+    public void testBenchmark()
+    {
+        setBenchmark("test_benchmark");
+        executionDriver.execute();
+        verifyBenchmark();
+    }
+
+    private static void startBenchtoService(Network network)
+    {
+        postgres = new PostgreSQLContainer<>("postgres:11")
+                .withNetwork(network)
+                .withNetworkAliases("postgres");
+        postgres.start();
+        String jdbcUrl = format("jdbc:postgresql://%s:%d/%s?user=%s&password=%s",
+                "postgres",
+                postgres.getExposedPorts().get(0),
+                postgres.getDatabaseName(),
+                postgres.getUsername(),
+                postgres.getPassword());
+        service = new GenericContainer<>("prestodev/benchto-service:latest")
```

Comment:
will local build with `maven` always be latest?

Reply:
Yes, maven docker plugin by default tags it with 'latest'.